stylelint 12.x as peer dependency

Hi,

stylelint [12.0.0](https://github.com/stylelint/stylelint/releases/tag/12.0.0) was released but do you have a plan to update the peer dependency?

https://github.com/YozhikM/stylelint-a11y/blob/1c522f677e1bd68420cc47a935e9865dd797159a/package.json#L40

I get the following error when installing `stylelint@12` and `stylelint-a11y`. 😢 

> npm ERR! peer dep missing: stylelint@^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0, required by stylelint-a11y@1.2.2

It would be great if you could consider it.
Thanks.

--- a/package.json
+++ b/package.json
@@ -37,7 +37,7 @@
     ]
   },
   "peerDependencies": {
-    "stylelint": "^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0"
+    "stylelint": "^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0"
   },
   "devDependencies": {
     "babel-cli": "^6.26.0",
@@ -58,7 +58,7 @@
     "lodash": "^4.17.11",
     "prettier": "^1.14.3",
     "rimraf": "^2.6.2",
-    "stylelint": "^11.1.1",
+    "stylelint": "^12.0.0",
     "stylelint-test-rule-tape": "^0.2.0"
   }
 }

